-------------------------------------------------------------------------------
Test set: com.finartz.RestApp.RestAppApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 5.381 s <<< FAILURE! - in com.finartz.RestApp.RestAppApplicationTests
contextLoads  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: mappedBy reference an unknown target entity property: com.finartz.RestApp.model.County.county in com.finartz.RestApp.model.City.countySet
Caused by: org.hibernate.AnnotationException: mappedBy reference an unknown target entity property: com.finartz.RestApp.model.County.county in com.finartz.RestApp.model.City.countySet

